### DISCUSSION TITLE
v0.2.0

### DISCUSSION THREAD

**Discussion by OXeu** [OXeu/Rin#39] [Announcements]
## 更新日志

- 支持 SEO 预渲染优化 #27 
- 支持时间轴归档 #23 
- 环境变量加密列表与变量列表更新，仅保留必须加密的环境变量
- 加密变量现在可以通过 Github 直接配置
- Github 变量配置更新，新增必须通过 Github 配置的加密变量（S3 存储，用于 SEO 索引保存）
- `GITHUB_CLIENT_ID`与`GITHUB_CLIENT_SECRET`现在添加了前缀`RIN_`（`RIN_GITHUB_CLIENT_ID`,`RIN_GITHUB_CLIENT_SECRET`），以解决 Github 变量不能以 `GITHUB_` 开头的问题，使用 Cloudflare 面板配置的 `GITHUB_CLIENT_ID` 与 `GITHUB_CLIENT_SECRET` 不受影响

## 迁移
1. 同步 fork 的仓库至最新代码
2. 参照[迁移文档](https://github.com/OXeu/Rin/blob/main/docs/DEPLOY.md)完成环境变量的迁移

## What's Changed
* docs: add issues template support by @OXeu in https://github.com/OXeu/Rin/pull/13
* docs: add Simplified Chinese CONTRIBUTING.md support. by @OXeu in https://github.com/OXeu/Rin/pull/14
* docs: Update README.md to use responsive image for Star History Chart by @OXeu in https://github.com/OXeu/Rin/pull/15
* fix: publish button shadow color on mobile view (dark mode) by @OXeu in https://github.com/OXeu/Rin/pull/16
* feat: Add pagination support by @OXeu in https://github.com/OXeu/Rin/pull/17
* feat: Optimize feed fetching logic and add support for different feed… by @OXeu in https://github.com/OXeu/Rin/pull/18
* fix: feed fetching logic and add support for different feed types by @OXeu in https://github.com/OXeu/Rin/pull/19
* fix: invalid page for spider by @OXeu in https://github.com/OXeu/Rin/pull/21
* docs: update README list numbering by @SliverKeigo in https://github.com/OXeu/Rin/pull/25
* fix: rename herf to href by @OXeu in https://github.com/OXeu/Rin/pull/26
* feat: seo friendly by @OXeu in https://github.com/OXeu/Rin/pull/27
* ci: Install Chrome for Puppeteer in SEO workflow by @OXeu in https://github.com/OXeu/Rin/pull/28
* fix: Fix link filtering logic in fetchPage function by @OXeu in https://github.com/OXeu/Rin/pull/29
* docs: Update SEO.md by @OXeu in https://github.com/OXeu/Rin/pull/30
* docs: add SEO.md to README.md by @OXeu in https://github.com/OXeu/Rin/pull/31
* docs: add migration guide by @OXeu in https://github.com/OXeu/Rin/pull/32
* docs: Update issue templates by @OXeu in https://github.com/OXeu/Rin/pull/33
* docs: add tips by @OXeu in https://github.com/OXeu/Rin/pull/34
* docs: add troubleshooting by @OXeu in https://github.com/OXeu/Rin/pull/35
* feat: add timeline support by @OXeu in https://github.com/OXeu/Rin/pull/37
* fix: can not add env start with `GITHUB_` by @OXeu in https://github.com/OXeu/Rin/pull/38

## New Contributors
* @SliverKeigo made their first contribution in https://github.com/OXeu/Rin/pull/25

**Full Changelog**: https://github.com/OXeu/Rin/compare/v0.1.0...v0.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/OXeu/Rin/releases/tag/v0.2.0'>v0.2.0</a>.</em>

**Comment by SliverKeigo** [comment]
🤩
